flowkit_jwt_generator/flowkit_jwt_generator/jwt.py

```diff
@@ -36,12 +121,14 @@ def generate_token(
         Lifetime from now of the token
     claims : dict
         Dictionary of claims the token will grant
+    audience : str, optional
+        Optionally provide a string to identify the audience of the token
```

Comment:
Hmm, upon reflection I wonder whether within the FlowKit context we should give this argument a more explicit name than "audience". I know this is the JWT terminology, but for a FlowKit user it is not obvious what "audience" refers to. How about naming it `flowapi_server_name` or `flowapi_target_server` or something similar? This would help clarify what should be passed here, and make the usage more intuitive. At the very least we should tweak the docstring to explain what "audience" means.